content/help/addons/hero-image.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Hero Image
+tags:
+- Addons
+- Hero Image
+- Image upload
+category: help
+menu:
+  help:
+    parent: addons
+    identifier: addon-hero-image
+versioning:
+  added: 2.6
+---
+
+## Overview
+
+{{% cloudfeature %}}
+
+This plugin provides an easy way to have a custom background image for every page of the forum. It allows you to upload this image in the dashboard and access it through a [Smarty Template](/developer/smarty/functions/#function-hero-image-url) or [Customize Theme](/help/appearance/custom-theme).
+
+It also allows you override an the main image on a per-category basis. Overriding this background image for a category will also override the background image for every category or discussion inside of that category. This allows you to have a tree of background images through your forum, managed through the dashboard.
+
+<aside class="note">In order to be flexible this plugin does __not__ provide an out of the box solution for existing themes. Integrating the image will require custom theming work to be done. It's best to be knowledgeable in both HTML and CSS before using this plugin.</aside>
+
+
+## Setup
+
+- Visit the addon's page of the dashboard. (`dashboard/settings/addons`)
+- Enable the HeroImage plugin.
+![](/img/help/addons/hero-image/hero-image-enable.png)
+
+## Set the hero image
+
+1. Click on the settings icon next to the plugin.
+![](/img/help/addons/hero-image/hero-image-settings.png)
+2. Click on the file picker to select your image.
+![](/img/help/addons/hero-image/hero-image-set-default.png)
+3. Click save.
+
+## Override the hero image for a category.
+
+1. Navigate to the category page of the dashboard. (`/vanilla/settings/categories`)
+2. Edit a category.
+3. Scroll to the Hero Image section.
+4. Use the file picker to upload a file.
+![](/img/help/addons/hero-image/hero-image-set-category.png)
+5. Click Save.
+
+## Which image will show up for a page?
+
+The default image is the one set in [plugin's setting](#set-the-hero-image). This image will always be returned for pages like Activity, Profile, Search, and many Ctegory & Discussion pages.
+
+A Discussion page will always return the same image as the category it is in.
+
+A Category page will by default return the default image unless:
```

Comment:
lower case category.

I wouldn't say "through the dashboard" because both the category images AND the default are through the dashboard. I would say: If no image is set on this category, it will look to it's parent(s) until it finds one with a image, or use the fallback image set through the plugin settings (anchor to "Set the hero image")